src/components/custom/zurich/GlobalNav.tsx

```diff
@@ -87,6 +87,15 @@ const HomeLink = styled.a`
         ${(props) => props.theme.spaces.s050} 0;
     }
   }
+
+  @media (min-width: ${(props) => props.theme.breakpointXl}) {
+    svg {
+      width: 242px;
+      margin: ${(props) => props.theme.spaces.s050}
+        ${(props) => props.theme.spaces.s150}
+        ${(props) => props.theme.spaces.s050} 0;
```

Comment:
Nitpick: This can be removed, the styles from the `breakpointMd` block will still be applied so there's no need to repeat these margin styles